### PR TITLE
chore: inter body font

### PR DESCRIPTION
**Pull request by Milouu**
## Description

Add Inter font and apply it to content body

### CODE REVIEW

File: src/components/DrawerSection.tsx [hunk 1]
```diff
@@ -248,6 +253,7 @@ export const PermissionsDrawerSectionEntry = ({
                 width="120px"
                 flexShrink="0"
                 paddingTop="2px" // compensates for the extra height of tag components
+                fontWeight="500"
```
Comment (oleobal) [inline src/components/DrawerSection.tsx:256]: Out of curiosity, why is this needed? What font does it default to?

Comment (Milouu) [reply src/components/DrawerSection.tsx:256]: Defaults to 400 :)

**Review by ThibaultFy — APPROVED** [review]
Thanks ! :)
